client/src/components/friends-page/FriendsContent.js

```diff
@@ -1,27 +1,21 @@
 import "./FriendsContent.css";
-
-import { getItemsFromLocalStorage } from "../../utils/itemStorage";
-import { useEffect, useState } from "react";
 import { GiSittingDog } from "react-icons/gi";
 import CloudRight from "./CloudRight";
 import CloudLeft from "./CloudLeft";
 import FakeFrens from "../../utils/FakeFrens";
 import CloudFormationsLeft from "../../utils/CloudFormationsLeft";
 import CloudFormationsRight from "../../utils/CloudFormationsRight";
+import useFriends from "../../hooks/useFriends";
 
 export default function FriendsContent() {
-  const [friends, setFriends] = useState([]);
-
-  useEffect(() => {
-    const friends = getItemsFromLocalStorage("friends");
-    setFriends(friends);
-  }, []);
+  const [friends, setFriends] = useFriends();
 
   function renderItems() {
     return friends.map((friend, index) => {
       if (index % 2 === 0) {
         return (
           <CloudRight
+            setFriends={setFriends} // where can this be used?
```

Comment:
what do you mean?
If you don't need setFriends you can just have your custom hook not return it

then it is just 
```
const friends = useFriends
```
in line 11 and in the other file

Reply:
Mind. Blown.